paddle/fluid/inference/tests/api/analyzer_dam_tester.cc

```diff
@@ -194,6 +194,8 @@ void profile(bool use_mkldnn = false) {
 
   if (use_mkldnn) {
     cfg.EnableMKLDNN();
+    std::unordered_set<std::string> op_list = {"conv3d"};
+    cfg.SetMKLDNNOp(op_list);
```

Comment:
please update the compare funcation as well.
```
void compare(bool use_mkldnn = false) {
  AnalysisConfig cfg;
  SetConfig(&cfg);
  if (use_mkldnn) {
    cfg.EnableMKLDNN();
  }
```

Reply:
Done, Thanks for your advice.